### PR TITLE
Add env setup page. Rename a few pages

### PR DESCRIPTION
**Pull request by sethkinast**

### CODE REVIEW

File: _includes/nav.md [hunk 1]
```diff
@@ -9,11 +9,10 @@
     <li class="sub">
       <input type="checkbox" name="nav-radio" id="guide-radio"><label for="guide-radio">Guides</label>
       <ul>
+      <li><a href="/guides/setup">Download &amp; Setup</a></li>
```
Comment (smfoote) [inline _includes/nav.md:12]: Do you think downloading should happen before getting started?

File: guides/setup.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## npm
+
+To install as a Node module:
+
+```
+npm install --save dustjs-linkedin
+```
+
+Dust should work with any version of Node or io.js newer than 0.8.12.
```
Comment (smfoote) [inline guides/setup.md:17]: io.js shout out... nice

File: guides/setup.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+The command-line compiler is located at `./node_modules/.bin/dustc`
```
Comment (smfoote) [inline guides/setup.md]: Maybe make a note about how to make the command line compiler globally available.

File: guides/setup.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+```
+bower install dustjs-linkedin
+```
+
+Dust is compatible with IE7+ and all modern browsers.
+
+## Download
+
+The latest version of Dust is available on the GitHub <a target="_blank" href="https://github.com/linkedin/dustjs/tree/master/dist">page</a>
+(or <a target="_blank" href="">past versions</a>).
```
Comment (smfoote) [inline guides/setup.md]: Do you not know where the past versions are kept?

File: guides/setup.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Include `dust-core.min.js` or `dust-full.min.js` on your page. If you will be compiling templates in the browser, use `dust-full.min.js`.
+
+## AMD
+
+As of Dust 2.5.2, you can load Dust using an AMD-compatible loader such as require.js.
+
+To maintain backwards compatibility, you must enable a config flag to tell Dust to register itself as a named AMD module ([in the same way jQuery works](http://requirejs.org/docs/jquery.html)).
+
+    define.amd.dust = true;
```
Comment (smfoote) [inline guides/setup.md:53]: Where does this go? Inside of `dust-core.min.js`?

File: _includes/nav.md (hunk 1 above)
Comment (sethkinast) [inline _includes/nav.md:12]: Yes. Getting started is the part where you learn to write your first templates and such. Should assume that you already have Dust ready to use.

File: guides/setup.md (hunk 3 above)
Comment (sethkinast) [inline guides/setup.md]: I think that's more of Node knowledge than Dust documentation, really.

You would know that something in .bin can be globalized with `-g`.

File: guides/setup.md (hunk 4 above)
Comment (sethkinast) [inline guides/setup.md]: No one cares about past versions and if you do you know enough to go find them yourself

File: guides/setup.md (hunk 5 above)
Comment (sethkinast) [inline guides/setup.md:53]: line 56 shows you an example

File: guides/setup.md (hunk 4 above)
Comment (sethkinast) [inline guides/setup.md]: (I fixed the link though :wink: oops)

File: guides/setup.md (hunk 3 above)
Comment (smfoote) [inline guides/setup.md]: Someone who knows Node would know this, but I don't think we should assume that all of our users know Node. Does it hurt to add the comment?

**Comment by smfoote** [comment]
Thanks!

**Pull request merged**